### PR TITLE
[14-add-model] Adding attributes to models and updating test_models

### PR DESCRIPTION
**Pull request by evpoe**
- Find/replace YourResourceName with Supplier
- Updated models.py table schema and serialization/deserialization
- Updated test_models.py per JR's video and made sure nosetests passed

Please note we did not have to update the requirements.txt like JR did in the video. We already had the necessary postgres libraries included.

### CODE REVIEW

File: service/models.py [hunk 1]
```diff
@@ -27,14 +27,17 @@ class YourResourceModel(db.Model):
 
     # Table Schema
     id = db.Column(db.Integer, primary_key=True)
-    name = db.Column(db.String(63))
+    name = db.Column(db.String(63), nullable=False)
+    email = db.Column(db.String(63), nullable=False)
```
Comment (sa3613) [inline service/models.py:31]: This looks like the right schema to me

File: service/models.py [hunk 2]
```diff
@@ -27,14 +27,17 @@ class YourResourceModel(db.Model):
...
+    address = db.Column(db.String(256), nullable=False)
+    phone_number = db.Column(db.String(63))
```
Comment (sa3613) [inline service/models.py:33]: I think this needs to be less characters, phone numbers are only 15 characters at most...evan...

Comment (sa3613) [inline service/models.py:33]: I think this needs to be less characters, phone numbers are only 15 characters at most...evan...

Comment (evpoe) [reply service/models.py:33]: Good suggestion, I will stick with 20 to be safe

**Pull request merged**